src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -542,7 +542,10 @@ export class SearchView extends ViewPane {
 		dom.append(excludesList, $('h4', undefined, excludesTitle));
 		this.inputPatternExcludes = this._register(this.instantiationService.createInstance(ExcludePatternInputWidget, excludesList, this.contextViewService, {
 			ariaLabel: excludesTitle,
-			placeholder: nls.localize('placeholder.excludes', "e.g. *.ts, src/**/exclude"),
+			// --- Start Positron ---
+			// placeholder: nls.localize('placeholder.excludes', "e.g. *.ts, src/**/exclude"),
+			placeholder: nls.localize('placeholder.excludes', "e.g. *.html, man/**/*.md"),
```

Comment:
```suggestion
			placeholder: nls.localize('placeholder.excludes', "e.g. *.html, man/**/*.Rd"),
```

Did you mean `.Rd` here?

Reply:
No, I meant `.md` for use like this: https://github.com/tidymodels/parsnip/tree/main/man/rmd

Reply:
Nit: I also *think* that `.Rd` files really do have to appear in the top-level of `man`, so we could consider a simpler pattern that still covers all (or most?) real-world situations.

Reply:
Ah, ok, re: `man/**/*.md`. Got it.

I do think it would be good to also feature `*.Rd` here, as that is that actual file type that most annoys me in search.